Automattic/wp-calypso
# Wrap lost password screen in a feature flag

#### Proposed Changes

* Add `login/react-lost-password-screen` feature flag
* Wrap lost password link and render component in the feature flag

#### Testing Instructions



1. Go to `https://your_local_calyso_host/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D2c3e8bc38c94c219730265dfbd807b4387046d4774f17e9480f8c833e9ac0fce%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login&client_id=50916&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Fref%3Doauth2%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%252F%253Fresponse_type%253Dcode%2526client_id%253D50916%2526state%253D2c3e8bc38c94c219730265dfbd807b4387046d4774f17e9480f8c833e9ac0fce%2526redirect_uri%253Dhttps%25253A%25252F%25252Fwoocommerce.com%25252Fwc-api%25252Fwpcom-signin%25253Fnext%25253Dmy-dashboard%2526blog_id%253D0%2526wpcom_connect%253D1%2526authorize%253D1%2526wccom-from%2526calypso_env%253Dproduction%2526jetpack-code%2526jetpack-user-id%253D0%2526action%253Doauth2-login%26oauth2_client_id%3D50916`
2. Click `Forgot password?`
3. Observe that you're redirected to react lost password screen
4. Set the feature flag to `false` in develpoment.json
5. Repeat 2~3
6. Observe that you're redirected to https://wordpress.com/wp-login.php?action=lostpassword
7. Repeat 2~3 with cookie `enable_react_password_screen=yes`
8.  Observe that you're redirected to react lost password screen


#### Pre-merge Checklist






Related to #
